modules/installation-network-user-infra.adoc

```diff
@@ -101,6 +101,8 @@ the Cluster Version Operator on port `9099`.
 |`10256`
 |openshift-sdn
 
+|`1936`
+|Metrics
```

Comment:
@sagidlow - I suggest making this the first cell in the `TCP` entry based on numerical ordering. WDYT?

Reply:
Sounds reasonable to me!

Reply:
@bobfuru Done :)